sdcm/tester.py

```diff
@@ -2621,51 +2602,6 @@ def copy_data_between_tables(self, node, src_keyspace, src_table, dest_keyspace,
         self.log.debug('All rows have been copied from %s to %s', src_table, dest_table)
         return True
 
-    def collect_partitions_info(self, table_name, primary_key_column, save_into_file_name) -> dict | None:  # pylint: disable=too-many-locals
```

Comment:
did you run LWT tests after this refactor?

Reply:
data validator was added for LWT. And here it was changed. So we need to be sure that LWT test is not broken

Reply:
@yarongilor did you?

Reply:
@roydahan , yes - https://github.com/scylladb/scylla-cluster-tests/pull/6185#issuecomment-1723549730

Reply:
Next time worth writing it and the original commenter should resolve it.